models/User.js

```diff
@@ -128,6 +131,19 @@ module.exports = function(sequelize, DataTypes) {
   return sequelize.define(name, attributes, options);
 };
 
+const changeGlobalPermission = async function(admin, user, permission) {
+  if (!user || !admin || !admin.isAdmin()) {
+    return false;
+  }
+  user.setDataValue('globalPermission', permission);
```

Comment:
I'm pretty sure (but not 100%) that you can just do: `user.globalPermission = permission;`